common/Resources/ti.internal/extensions/ti/ti.ui.view.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* globals OS_ANDROID,OS_IOS */
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/ti/ti.ui.view.js;
  Line: 7;
-->

- :warning: common/Resources/ti.internal/extensions/ti/ti.ui.view.js line 7 – 'OS_IOS' is defined but never used. (no-unused-vars)